docs/adopted-rules.md

```diff
@@ -0,0 +1,122 @@
+# Adopted Rules
+
+## Weapon and Armor Levels
+(*Source: [Fate Core](https://fate-srd.com/fate-core/more-examples-extras#weapon-and-armor-ratings)*)
+
+Physical weapons and armor are not typically tracked as resources in Fate. 
+
+Instead, "Weapon" and "Armor" refer to a type of Extra that a character can have. Taking Weapon or Armor Extra grants a passive modifier to damage calculation. Weapon adds to damage dealt, and Armor reduces damage taken. The modifier is equal to the rating Level. 
```

Comment:
Referring to it as an extra is likely to get confusing. We should treat weapon and armor ratings as a tool in our toolbelt to make Stunts more interesting rather than something that has to be defined separately.